go.mod

```diff
@@ -1,21 +1,27 @@
 module github.com/ShiftLeftSecurity/atlassian-connect-go
 
-go 1.13
+go 1.17
```

Comment:
.18?

Reply:
I wasn't sure we wanted to drop support for .17 yet, but I don't mind :-)

Reply:
Fair